### PR TITLE
[android] Added missed events for crown btn

### PR DESCRIPTION
**Pull request by yoksnod**
https://jira.mail.ru/browse/MAPSME-11895
https://jira.mail.ru/browse/MAPSME-11896

### CODE REVIEW

File: android/src/com/mapswithme/util/statistics/Statistics.java [hunk 1]
```diff
@@ -184,6 +184,12 @@ public static ParameterBuilder makeDownloaderBannerParamBuilder(@NonNull String
             .add(PROVIDER, provider);
   }
 
+  public static ParameterBuilder makeGuidesSubscriptionBuilder()
```
Comment (alexzatsepin) [inline android/src/com/mapswithme/util/statistics/Statistics.java]: @nonnull

File: android/src/com/mapswithme/util/statistics/Statistics.java [hunk 2]
```diff
@@ -693,6 +702,7 @@ private EventParam() {}
     public static final String SPONSORED_BUTTON = "sponsored_button";
     public static final String POPUP = "popup";
     public static final String WEBVIEW = "webview";
+    public static final String GUIDES_SUBSCRIPTION = "GuidesSubscription";
```
Comment (alexzatsepin) [inline android/src/com/mapswithme/util/statistics/Statistics.java]: package private

File: android/src/com/mapswithme/util/statistics/Statistics.java [hunk 3]
```diff
@@ -598,6 +606,7 @@ private EventName() {}
     public static final String SETTINGS = "settings";
     public static final String ROUTE = "route";
     public static final String SCENARIO = "scenario";
+    public static final String TARGET = "target";
```
Comment (alexzatsepin) [inline android/src/com/mapswithme/util/statistics/Statistics.java]: package private

**Comment by yoksnod** [comment]
@alexzatsepin PTAL

**Pull request merged**